### PR TITLE
Adding Endpoint Print func

### PR DESCRIPTION
**Pull request by gerbsec**

### CODE REVIEW

**Review by bfuzzy1 — APPROVED** [review]
@gerbsec can you update all the tests in tests/ master to use this new print function please?

Approving since we tested this together.

**Comment by bfuzzy1** [comment]
@privateducky would like your approval on this change as well. Also we should probably hold back on merging this until the swap for real test ids goes in this week.

File: go/tests/endpoint/endpoint.go [hunk 1]
```diff
@@ -44,7 +44,7 @@ func Stop(code int) {
 func Print(print string) {
 	filename := filepath.Base(os.Args[0])
 	name := strings.TrimSuffix(filename, filepath.Ext(filename))
-	Print(fmt.Sprintf("[%s] %s\n", name, print))
+	fmt.Sprintf("[%s] %s\n", name, print)
```
Comment (privateducky) [inline go/tests/endpoint/endpoint.go]: check out ``fmt.Sprintf("[%s] %v", name, print)``

File: go/tests/endpoint/endpoint.go [hunk 2]
```diff
@@ -23,7 +23,7 @@ func Start(test fn, clean ...fn) {
 		cleanup = clean[0]
 	}
 
-	Print("Starting test")
+	Print(fmt.Sprintf("Starting test at: %s", time.Now().UTC().Format("2006-01-02T15:04:05Z")))
```
Comment (privateducky) [inline go/tests/endpoint/endpoint.go:26]: we should do local time

Comment (bfuzzy1) [reply go/tests/endpoint/endpoint.go:26]: I fix.

Comment (bfuzzy1) [reply go/tests/endpoint/endpoint.go]: fixed!

**Review by privateducky — APPROVED** [review]

**Pull request merged**